solidity/dashboard/src/components/DelegatedTokensTable.jsx

```diff
@@ -5,10 +5,11 @@ import UndelegateStakeButton from './UndelegateStakeButton'
 import StatusBadge, { BADGE_STATUS } from './StatusBadge'
 import { PENDING_STATUS, COMPLETE_STATUS } from '../constants/constants'
 import { DataTable, Column } from './DataTable'
+import Tile from './Tile'
 
 const DelegatedTokensTable = ({ delegatedTokens, cancelStakeSuccessCallback }) => {
   return (
-    <section className="tile">
+    <Tile className="tile">
```

Comment:
Just wanted to make sure that we don't need to add <LoadingOverlay> here?

Reply:
yep. I'll update